app/code/Magento/Newsletter/Test/Mftf/ActionGroup/AdminDisConfNewsletterEnableRegConfirmActionGroup.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminDisConfNewsletterEnableRegConfirmActionGroup">
+        <amOnPage url="{{AdminCustomerConfigPage.url('#customer_create_account-link')}}" stepKey="goToCustomerConfig"/>
+        <waitForPageLoad stepKey="waitForCustomerAdminPageLoad"/>
+        <conditionalClick selector="{{AdminStoresCustomerConfigurationSection.createNewAccOpt}}" dependentSelector="{{AdminCustomerConfigSection.customerCreateAccountCheckbox}}" visible="false" stepKey="click1"/>
```

Comment:
You don't need this since you use #customer_create_account-link anchor that open this section automatically